numpy/core/numeric.py

```diff
@@ -692,17 +692,27 @@ def correlate(a, v, mode='valid'):
 
     Notes
     -----
-    The definition of correlation above is not unique and sometimes correlation
-    may be defined differently. Another common definition is:
+    The definition of correlation above is used to implement the correlate function.
+    This definition is not unique and sometimes correlation
+    may be defined differently. Another common definition is [2]_ :
 
     .. math:: c'_k = \sum_n a_{n} \cdot \overline{v_{n+k}}
 
-    which is related to :math:`c_k` by :math:`c'_k = c_{-k}`.
+    :math:`c_k` is equivalent to :math:`c_{-k}` by :math:`c'_k = c_{-k}`.
```

Comment:
Mathematically or computationally equivalent? See my comment here: https://github.com/numpy/numpy/issues/23782#issuecomment-1556158295

Reply:
I have updated the documentation to state c'_k is mathematically and computationally equivalent to c_{-k}